src/explorer/SiteTreeItem.ts

```diff
@@ -49,6 +49,16 @@ export abstract class SiteTreeItem implements IAzureParentTreeItem {
         return this.site.id;
     }
 
+    public async start(client: WebSiteManagementClient): Promise<void> {
+        await this.siteWrapper.start(client);
+        await this.reloadSite(client);
+    }
+
+    public async stop(client: WebSiteManagementClient): Promise<void> {
+        await this.siteWrapper.stop(client);
+        await this.reloadSite(client);
+    }
+
     public browse(): void {
```

Comment:
Currently if a web app is Stopped and then you "Restart" the app through the context menu, the state doesn't refresh unless you manually refresh the node.

Reply:
Yep, I'll handle that case.